deps/simdutf/simdutf.gyp

```diff
@@ -0,0 +1,27 @@
+{
+  'targets': [
+    {
+      'target_name': 'simdutf',
+      'type': 'static_library',
+      'include_dirs': ['.'],
+      'sources': ['simdutf.cpp'],
+      'cflags': [ '-Wno-unused-function' ],
+      'conditions': [
+        [ 'OS!="win"', {
+          'cflags': [ '-mavx' ],
```

Comment:
Forcing this here and similarly below is not going to work. Some architectures don't have AVX and as far as x86 goes, I believe node(/V8?) still supports back to SSE 4.2.

I think it would be better to do it like the base64 and zlib dependency gyp files to ensure compatibility is maintained.

Reply:
Thanks for the feedback. I think I got the idea and updated the pull request. Appreciate it if you could take a look at it @mscdex